### PR TITLE
chore(website): remove local blog in favour of the DevBlog

### PR DESCRIPTION
**Pull request by kelset**
## Description & Why

hey there - as mentioned a few times over the last few weeks, now that we have the [official devblog up and running](https://devblogs.microsoft.com/react-native/) we will move all the blogpost content over there - the blogposts from here have already been migrated over.

This PR takes care of redirecting the links to it; for the single posts, it's using the only alternative we could find: using the [HTTP redirect](https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections#html_redirections) via [Head Metadata](https://docusaurus.io/docs/markdown-features/head-metadata).

While doing so I've also done some small cleanups + a couple fixes that were causing CI to fails

## Screenshots

Nothing really, visually it looks the same, simply clicking on blogpost will send over to the devblog; you can also try to put the direct url to a single blogpost and it will redirect to the new place.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows-samples/pull/746)

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -285,18 +268,6 @@ class Index extends React.Component {
...
-      <Section background="light">
-        <div className="content">
-          <Heading text="Status and Roadmap" />
-          <h2>Windows</h2>
-          <MarkdownBlock>{textContent.roadmapwindows}</MarkdownBlock>
-          <h2>macOS</h2>
-          <MarkdownBlock>{textContent.roadmapmac}</MarkdownBlock>
-        </div>
-      </Section>
-    );
```
Comment (kelset) [inline website/pages/en/index.js:298]: I removed this since it was unused and outdated

**Comment by asklar** [comment]
@kelset  note that we have several aka.ms links that point to blogs, have we migrated those too?
e.g. https://aka.ms/rnwinrtblog

**Comment by kelset** [comment]
> @kelset Lorenzo Sciandra FTE note that we have several aka.ms links that point to blogs, have we migrated those too? e.g. [aka.ms/rnwinrtblog](https://aka.ms/rnwinrtblog)

it's a good question, technically if we figure out a way to do the redirects proper, it shouldn't be necessary: aka.ms will go to the old url -> old url redirects to devblog url -> profit.

Problem is figuring out how to do these redirects, we are trying to figure out a way via html redirects as they seem to be the only thing working with vanilla GH pages hosting

**Comment by kelset** [comment]
update: we figured out how to do the redirects, please check out the updated PR body for details. PR is now ready to go @stmoy @asklar

File: docs/supported-community-modules.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ In addition to the ejected modules above, Microsoft has also added implementatio
 | <ins>[`rn-fetch-blob`](https://github.com/joltup/rn-fetch-blob)</ins> | [0.62](https://github.com/joltup/rn-fetch-blob/pull/701) | x |
 | <ins>[`react-native-code-push`](https://github.com/Microsoft/react-native-code-push)</ins> | 0.63 | x |
 | <ins>[`react-native-tts`](https://github.com/ak1394/react-native-tts)</ins> | 0.62 | x |
-| <ins>[`react-native-sketch-canvas`](https://github.com/creambyemute/react-native-sketch-canvas)</ins> | 0.63 | x |
+| <ins>[`react-native-sketch-canvas`](https://github.com/wwimmo/react-native-sketch-canvas)</ins> | 0.63 | x |
```
Comment (asklar) [inline docs/supported-community-modules.md:59]: is this change intentional?

**Review by asklar — APPROVED** [review]
lgtm other than the canvas repo thing

File: docs/supported-community-modules.md (hunk 2 above)
Comment (kelset) [reply docs/supported-community-modules.md:59]: yes, CI was broken because it was trying to go to the old repo but that link redirects to this new one; the "link checker" failed/doesn't handle redirects so it went ❌

you can verify the behaviour by clicking here https://github.com/creambyemute/react-native-sketch-canvas

Comment (asklar) [reply docs/supported-community-modules.md:59]: gotcha thanks!
